### ISSUE TITLE
Template gitignore file should exclude binlogs

### ISSUE THREAD

**Issue by NickGerleman** [microsoft/react-native-windows#5937]
We started dropping binlog files in the project root directory during build. These aren't included in the default gitignore and will be checked in by default.

**Comment by gat786** [comment]
Hey can I take this issue and work on fixing it?

**Comment by acoates-ms** [comment]
Go for it.

**Comment by gat786** [comment]
Hey @acoates-ms can you guide me through what must be done here?
From what I understand the problem is just I need to remove the `.binlog` files from `.gitignore` file.

**Comment by acoates-ms** [comment]
No you need to add the binlog files to the git ignore file that is created during react-native-windows-init. 
I believe this file https://github.com/microsoft/react-native-windows/blob/master/vnext/template/_gitignore

**Comment by gat786** [comment]
was that it?

**Comment by NickGerleman** [comment]
I think that gitignore file might be for the windows directory instead of the root directory where we currently place the binlog (though I think the windows directory might be a better home for it).

If you wanted to validate, the steps would be:
1. Use the documentation in our Github Wiki on how to test the CLI to make a new RNW app with your changes
2. Init a new git repo in the created directory
3. Do a build with run-windows 
4. Confirm Git doesn't try to add binlogs

**Comment by gat786** [comment]
I tried building it with the changes I made. but it seems it still drops `.binlog` files in and isn't ignored by the `.gitignore` files

**Comment by goatandsheep** [comment]
try adding `*.binlog`

**Comment by goatandsheep** [comment]
@gat786 I had the same issue using Visual Studio. I'm not sure why

**Comment by gat786** [comment]
Well I think this needs to be fixed on the react native template gitignore sides of things rather than rnw template gitignore. I will check if that's the case  and leave an update.

**Comment by TatianaKapos** [comment]
Yep, looks like ".binlog" has to be in the react-native core template to catch binlog files since they appear in the root project directory instead of the windows folder.

Would the next steps be to figure out how to add ".binlog" to the react-native core template through _npx react-native-windows-init --overwrite_ or ask react-native core to add it to their template? @chrisglein
